pkg/build/inline.go

```diff
@@ -237,6 +243,10 @@ func (n *Inliner) objectFiles(ctx context.Context, objFiles []bundle.File, ref b
 				if err != nil {
 					return nil, nil, fmt.Errorf("converting multi-doc object number %d for component %v, %v", i, ref, err)
 				}
+				// It's easiest to extract the ObjectTemplateBuilder objects here,
+				// because, if relative paths are used within the
+				// ObjectTemplateBuilder, they'll be relative to the path used for the
+				// ObjectTemplateBuilder
```

Comment:
Side: we should figure out how to preserve location information through the expansion process.

Reply:
Oh probably. I used to have it and thought it was unnecessary, so I removed it. I could preserve it as an annotation.